oss-licenses-plugin/build.gradle

```diff
@@ -10,7 +10,7 @@ dependencies {
 }
 
 group = 'com.google.android.gms'
-version = '0.9.5'
+version = '0.9.6'
```

Comment:
Should this be 0.10.0 given changes in https://github.com/google/play-services-plugins/pull/87 which changes behavior of license collection?

Reply:
Feels like more of a bug fix to me, since the code originally seemed to be trying to handle the case by iterating over licenses in the first place, but was choosing the wrong XML element (the `<licenses>` wrapper element rather than the repeated `<license>` elements). But I don't feel strongly either way.

Reply:
Are we looking at the same change? https://github.com/google/play-services-plugins/pull/87 has the potential to exclude some license (e.g. any from `test` or `testCompile` deps) which would impact app behavior. I think that warrants a minor bump.

Reply:
Ah shoot I was only thinking of #88. Updated.